pages/design-tokens/color/overview.md

```diff
@@ -203,7 +203,7 @@ If we use color intentionally, consistently, and sensitively, it can make a big
 - W3C: Contrast levels for readability
   [https://www.w3.org/WAI/test-evaluate/preliminary/#contrast](https://www.w3.org/WAI/test-evaluate/preliminary/#contrast){:.display-block}
 - W3C: Gap analysis of the state of accessibility for people with learning disabilities and cognitive disabilities
-  [https://w3c.github.io/wcag/coga/gap-analysis.html](https://w3c.github.io/wcag/coga/gap-analysis.html){:.display-block}
+  [https://www.w3.org/TR/coga-gap-analysis/](https://www.w3.org/TR/coga-gap-analysis/){:.display-block}
```

Comment:
> [!TIP]
> ![image](https://github.com/user-attachments/assets/21afb602-f3f8-4bb6-9958-d151b9dcd4a9)
> [Preview link →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-w3-gap-analysis-link/design-tokens/color/overview/#further-reading-2:~:text=W3C%3A%20Gap%20analysis,coga%2Dgap%2Danalysis/)

---

**suggestion (non-blocking):** If the link text is the same as the URL you can use a shorthand.

```suggestion
  <https://www.w3.org/TR/coga-gap-analysis>{:.display-block}
```

Reply:
I'm going to leave as-is only because the rest of our links do not use this shorthand. But it's a good technique!